modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/pages/Project/Overview/components/SubscriptionsOverview/components/AccountSubscriptionsList/components/AccountSubscriptionModal/utils/getColumns.jsx

```diff
@@ -107,23 +107,23 @@ export default function getColumns(
 
 	const displayColumns = [...columns];
 
-	let displayInstanceSizeForProduct = false;
-	const displayInstanceSizeByCategories = [
-		'DXP',
+	let displayInstanceSizeForProduct = [
 		'Commerce',
+		'Liferay Self-Hosted',
 		'Portal',
-	].some((category) => title.startsWith(category));
+	].some((category) => {
+	    if (title.startsWith(category)) {
+	        const productName = title.substring(category.length + 1);
 
-	if (displayInstanceSizeByCategories) {
-		const [category, ...product] = title?.split(' ');
-		const productName = product.join(' ');
+	        return displayInstanceSizeMap[
+            	category
+            ].includes(productName);
```

Comment:
lines after return should be indented

Reply:
oh hm...formatsource doesn't seem to be working. I'll look into this tomorrow

Reply:
Going to create another ticket for this. formatsource is not checking our dir